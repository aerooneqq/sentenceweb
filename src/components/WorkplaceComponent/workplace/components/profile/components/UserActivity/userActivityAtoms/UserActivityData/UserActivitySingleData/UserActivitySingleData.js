import React, {Component} from "react";

export default class UserActivitiesSingleData extends Component