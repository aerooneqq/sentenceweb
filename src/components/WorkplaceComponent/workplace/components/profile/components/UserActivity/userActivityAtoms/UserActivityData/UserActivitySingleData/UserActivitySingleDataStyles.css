.singleActivityDataContainer { 
    width: 100%;
    height: auto;

    box-sizing: border-box;

    padding-left: 5%;
    padding-right: 5%;

    display: flex;
    flex-direction: row;
    justify-content: left;
    align-items: center;
    margin-top: 5px;
}

.singleActivityDataIcon { 
    width: 20px;
    height: 20px;
}

.singleActivityDataIcon { 
    width: 20px;
    height: 20px;
}

.activityInfoContainer { 
    display: flex;
    flex-direction: column;

    margin-left: 5px;
}

.singleActivityName { 
    font-size: 15px;
    color: #303030;
}

.singleActivityTime { 
    font-size: 11px;
    color: #404040;

    margin-top: 3px;
}

