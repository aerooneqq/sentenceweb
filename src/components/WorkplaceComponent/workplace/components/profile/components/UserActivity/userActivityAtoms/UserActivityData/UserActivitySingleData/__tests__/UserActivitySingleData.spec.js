import React from "react"; 
import {shallow} from "enzyme"; 
import {shallowToJson} from "enzyme-to-json";
import UserActivitySingleData from "../UserActivitySingleData";

const props = { 
    activity: { 
        activity: "test name",
        activityDate: "30.05.2000T23131231" 
    }
}

describe("UserActivitySingleData", () => {
    it("should render correctly", () => { 
        expect(shallowToJson(shallow(<UserActivitySingleData {...props}/>))).toMatchSnapshot();
    });

    it("should corectly display name", () => { 
        const userActivitySingleData = shallow(<UserActivitySingleData {...props}/>);
        expect(userActivitySingleData.find("div.singleActivityName").text()).toBe(props.activity.activity);
    });

    
});