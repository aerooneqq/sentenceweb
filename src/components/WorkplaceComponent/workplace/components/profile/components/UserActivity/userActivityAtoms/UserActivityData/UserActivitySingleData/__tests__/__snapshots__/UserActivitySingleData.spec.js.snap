// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserActivitySingleData should render correctly 1`] = `
<div
  className="singleActivityDataContainer"
>
  <div
    className="singleActivityDataIconCont"
  >
    <img
      alt="Activity icon"
      className="singleActivityDataIcon"
      src="single_user_activity_data_icon.png"
    />
  </div>
  <div
    className="activityInfoContainer"
  >
    <div
      className="singleActivityName"
    >
      test name
    </div>
    <div
      className="singleActivityTime"
    >
      23131231
    </div>
  </div>
</div>
`;
